scripts/perftest/python/utils_misc.py

```diff
@@ -23,15 +23,74 @@
 from os.path import join
 import os
 import json
-import subprocess
-import shlex
 import re
-import logging
 import sys
-import glob
-from functools import reduce
+from utils_exec import subprocess_exec
 
-# This file contains all the utility functions required for performance test module
+# This file contains all misc utility functions required by performance test module
+
+
+def sup_args(config_dict, spark_dict, exec_type):
```

Comment:
Not sure what this name means. Could you please think of a more descriptive name?